hello-world/nodejs-cli-app/start-instance.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Sample code to start the given instance based on instance OCID and region.
+ * 
+ * Sample Output:
+ * (Prints the instance details)
+ */
+
+const common = require("oci-common");
+const core = require("oci-core");
+
+const adp = new common.ConfigFileAuthenticationDetailsProvider();
+
+const region = prcess.env.REGION ? prcess.env.REGION : common.Region.US_ASHBURN_1;
```

Comment:
Is this a typo?  Shouldn't it be `process.env.REGION`?

Reply:
Yes. it's typo. Corrected now.